pkgs/applications/networking/cluster/terragrunt/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildGoModule rec {
   pname = "terragrunt";
-  version = "0.38.9";
+  version = "0.39.0";
 
   src = fetchFromGitHub {
     owner = "gruntwork-io";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-rfAVgnFAxEguFuY+Gfe/T0NcsD6LmPSquxuTR0bRqXQ=";
+    sha256 = "sha256-Y8t5rThkPD3FzY25L2lOIvktU5+NwBZtq9Pn3VzQQf4=";
   };
 
   vendorSha256 = "sha256-CqImT90jFFLi6XR7jfzFKwhnCHK6B+aM+Ba/L+G3bEg=";
```

Comment:
vendor has was not updated

Reply:
> vendor has was not updated

https://github.com/gruntwork-io/terragrunt/compare/v0.38.9...v0.39.0

There hasn't been any changes to the modules?

Reply:
Okay, thanks for checking.